core/src/commonMain/kotlin/com/xebia/functional/xef/llm/Chat.kt

```diff
@@ -17,6 +17,15 @@ interface Chat : LLM {
 
   fun tokensFromMessages(messages: List<Message>): Int
 
+  @AiDsl
+  suspend fun promptMessage(
```

Comment:
Isn't this conflicting with the function of the same name returning a `List`? My experience is that overloading by return type usually leads to problems, maybe we can think of another name?

Reply:
I found no conflicts, probably because all the use sites had type descriptions. 
Just in case, to address this I changed in [22c8a1e](https://github.com/xebia-functional/xef/pull/226/commits/22c8a1e8138a070724cd3d2e84252c6727e36a87) `promptMessage` to return a `String` and `promptMesages` to return a `List<String>` so there are no conflicts.